src/Base/Console.h

```diff
@@ -25,23 +25,20 @@
 #define BASE_CONSOLE_H
 
 // Std. configurations
+#include <array>
 #include <chrono>
 #include <map>
 #include <set>
 #include <string>
 #include <sstream>
 #include <FCGlobal.h>
 
+#include <fmt/printf.h>
+
 // Python stuff
 using PyObject = struct _object;
 using PyMethodDef = struct PyMethodDef;
 
-//FIXME: ISO C++11 requires at least one argument for the "..." in a variadic macro
-#if defined(__clang__)

```

Comment:
Unfortunately, this block (and the corresponding pop at the end) shouldn't be dropped because on MSYS2 clang raises this warning for each file that includes this header.

Reply:
Ok. 
I removed it because it was referring to the "..." in a variadic macro and now there is only ... as part of parameter packs. 

I will reinstate it.